tools/ClangTidyAll.sh

```diff
@@ -43,10 +43,10 @@ EXIT_CODE=0
 # Check for run-clang-tidy, then run-clang-tidy-LLVM_VERSION, and
 # if we can't find those, then just loop over the files one at a time.
 RAN_CLANG_TIDY=no
-for run-clang-tidy in run-clang-tidy run-clang-tidy.py "${RUN_CLANG_TIDY_BIN}" ;
+for run_clang_tidy in run-clang-tidy run-clang-tidy.py "${RUN_CLANG_TIDY_BIN}" ;
 do
-    if command -v "${run-clang-tidy}" > /dev/null 2>&1; then
-        ${run-clang-tidy} -quiet ${THREADING_FLAG} -p ${BUILD_DIR} \
+    if command -v ${run_clang_tidy} > /dev/null 2>&1; then
+        ${run_clang_tidy} -quiet ${THREADING_FLAG} -p ${BUILD_DIR} \
                           ${ALL_SOURCE_FILES} || EXIT_CODE=1
         RAN_CLANG_TIDY=yes
         break
```

Comment:
While you're here, there's a `$FILENAME` that should be a `$file` a few lines below this hunk.

Reply:
I switched to `FILENAME` in the three places in an attempt to make the `Hash` and `All` files be more consistent